spk/python312/src/requirements-pure.txt

```diff
@@ -1,28 +1,28 @@
 # basic default wheels
 # ==> Always install latest version (service-setup.sh:install_python_virtualenv)
-pip==24.3.1
-setuptools==78.1.0
+#pip==25.0.1    # we use module ensurepip to install latest version included
+setuptools==80.4.0
 wheel==0.45.1
 
 # Always use latest version available
-certifi==2025.1.31
+#certifi==2025.4.26
+certifi
```

Comment:
I thought @th0ma7 indicated that we should hardcode versions for the build so that the output is predictable? This is why I started changing to specific versions of `certifi` with the last update.

Reply:
certify is an exception as it is really the one that doesn't add any value to enforce a version since we always want the latest version no matter what.